frontend/src/metabase/query_builder/components/view/View.jsx

```diff
@@ -295,7 +301,13 @@ class View extends Component {
             mode={queryMode}
           />
         </StyledDebouncedFrame>
-        <TimeseriesChrome {...this.props} className="flex-no-shrink" />
+        {queryBuilderMode === "view" && (
```

Comment:
Should we call `renderMain` only when it's "view"?

Reply:
at least tests passed [10e689a](https://github.com/metabase/metabase/pull/39441/commits/10e689a2a1c20dc5fc294a9f57072e89b34a3bbd)